build/dependencies.props

```diff
@@ -8,6 +8,7 @@
     <MicrosoftSourceLinkGitHubVersion>1.0.0</MicrosoftSourceLinkGitHubVersion>
     <NetStandardVersion>2.0.3</NetStandardVersion>
     <NewtonsoftVersion>13.0.3</NewtonsoftVersion>
+    <SystemDiagnosticSourceVersion>9.0.0</SystemDiagnosticSourceVersion>
```

Comment:
We should think about if we want to use 9.x versions or not. 9.x libraries are going to be standard term support, vs 8.x libraries are going to be lts.

Reply:
it's close, but it looks like 8.x versions will have a few more months of support and I bet we'll want them. 8.x it is! good point